pkg/apis/hive/v1/agent/status.go

```diff
@@ -0,0 +1,44 @@
+package agent
+
+// InstallStrategyStatus defines the observed state of the Agent install strategy for this cluster.
+type InstallStrategyStatus struct {
+
+	// AgentNetworks provides an aggregate view of all network CIDRs detected across all Agents.
```

Comment:
> aggregate view

what type of aggregation is happening to agent (its ip address and network) and one item in this list? 
Is it just a 
- list of each agent and it ip/network or 
- list of aggregated newtorks and agents (more than one) in that network.

i think we should probably clarify the expectations here.

Reply:
Need your help here @avishayt @filanov @mhrivnak , could one of you send me some godoc text for this?